tests/ut/ir/transforms/test_paged_gather.py

```
@@ -0,0 +1,208 @@
+    # Static buffer (max_indices), dynamic loop bound (ROWS).
+    assert "pl.tile.create([64, 128]" in text
+    assert "ROWS" in text
+
+
+@pytest.mark.parametrize("is_trans", [False, True])
+def test_paged_gather_survives_full_pipeline(is_trans):
+    """The lowered loop survives the full Default pipeline through codegen lowering."""
+    try:
+        set_backend_type(BackendType.Ascend910B)
+    except Exception:
+        pass  # A backend may already be configured by an earlier test in the session.
```

Ruff (0.15.17)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)